### PR TITLE
Search button

### PR DESCRIPTION
**Pull request by FlorianBartier**
Ajout du bouton de recherche

### CODE REVIEW

**Review by Fbouchezpro — APPROVED** [review]
Review code qnd resolve conflict

File: package.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "styled-components": "^5.3.6"
```
Comment (LoicBrassart) [inline package.json]: I still don't want styled-components here :p

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One Minor Evil

**Review by Fbouchezpro — APPROVED** [review]
All ok for me

**Review by Fbouchezpro — APPROVED** [review]
Styled-components well uninstall

**Review by LoicBrassart — APPROVED** [review]
Much better !

**Pull request merged**